src/Core/TileMesh.js

```diff
@@ -85,7 +85,9 @@ TileMesh.prototype.changeState = function changeState(state) {
 };
 
 TileMesh.prototype.setFog = function setFog(fog) {
-    this.material.setFogDistance(fog);
+    if (fog != undefined) {
```

Comment:
In which situation would `setFog(...)` would be called with `fog = undefined` (ie: are you sure the bug isn't elsewhere?)

Reply:
Because the fogDistance isn't always defined, we can give it a default value. 
it's reason break on planar View

Reply:
Could you instead fix the caller to not call `setFog` with an undefined value please?

Reply:
done